src/api/json/catalog.json

```diff
@@ -9,6 +9,20 @@
       "fileMatch": [ ".angular-cli.json", "angular-cli.json" ],
       "url": "https://raw.githubusercontent.com/angular/angular-cli/master/packages/@angular/cli/lib/config/schema.json"
     },
+    {
+      "name": "Ansible",
+      "description": "Ansible task files",
+      "url": "http://json.schemastore.org/ansible-stable-2.5",
+      "fileMatch": [ "tasks/*.yml", "tasks/*.yaml" ],
```

Comment:
Not sure about the semantics of `fileMatch` here, is it really file matching or rather path matching?